resources/assets/js/app/views/dashboard/DashboardLayout.js

```diff
@@ -0,0 +1,30 @@
+var Marionette = require('backbone.marionette');
+var logger = require('../../instances/logger');
+var _ = require('underscore');
+var CountersCollectionView = require('./CountersCollection');
+var DashBoardProfileView = require('./DashBoardProfile');
+var DashBoardMainView = require('./DashBoardMain');
```

Comment:
var DashBoardMainView = require('./DashBoardMain');

DashBoardMain - please check filename and file that you requireing